CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - Fixed graphical glitches when resizing on Wayland.
 - On Windows, fix freezes when performing certain actions after a window resize has been triggered. Reintroduces some visual artifacts when resizing.
+- Updated window manager hints under X11 to v1.5 of "Extended Window Manager Hints" ([see here](https://specifications.freedesktop.org/wm-spec/wm-spec-1.5.html#idm140200472629520))
```

Comment:
I'd prefer if "Extended Window Manager Hints" were the link text (instead of "see here") with no quotation marks, and if it ended in a period.